cdc-accelerator/feature/policies.js

```diff
@@ -28,6 +29,8 @@ export default class Policies extends SiteFeature {
             throw new Error(JSON.stringify(policiesResponse))
         }
 
+        removePropertyFromObjectCascading(policiesResponse, 'confirmationEmailTemplates')
```

Comment:
We should not delete from the original response as it can cause unexpected behavior. Tests 'policies file does not contain unwanted fields' and 'policies file is generated successfully' are just passing due to this side effect.
Clone the response first and then use the cloned response, e.g:

async init(apiKey, siteConfig, siteDirectory) {
        const toolkitPolicies = new ToolkitPolicies(this.credentials, apiKey, siteConfig.dataCenter)
        const policiesResponse = await toolkitPolicies.get()
        if (policiesResponse.errorCode) {
            throw new Error(JSON.stringify(policiesResponse))
        }

        const filteredPoliciesResponse = JSON.parse(JSON.stringify(policiesResponse))
        removePropertyFromObjectCascading(filteredPoliciesResponse, 'confirmationEmailTemplates')
        removePropertyFromObjectCascading(filteredPoliciesResponse, 'emailTemplates')
        const featureDirectory = path.join(siteDirectory, this.getName())
        this.createDirectory(featureDirectory)

        // Create policy file
        this.removeResponseStatusFields(filteredPoliciesResponse)
        fs.writeFileSync(path.join(featureDirectory, Policies.POLICIES_FILE_NAME), JSON.stringify(filteredPoliciesResponse, null, 4))
    }

Then both tests will fail.
On the first because of this call this.removeResponseStatusFields(filteredPoliciesResponse) that is not being expected on the test.
The second test will fail because the writeFile was called with a payload containing the templates. The test 'policies file is generated successfully' can be deleted because it is testing the same as the new test 'policies file does not contain unwanted fields'